scss/example-15.scss

```diff
@@ -0,0 +1,46 @@
+$baseline-impl: (
+  grid-mq-small: 539px,
+  grid-mq-medium: 1039px,
+  columns-large: 15,
+  columns-medium: 6,
+  columns-small: 3,
+  grid-zones: (15, 12, 9, 6, 3),
+  grid-block: (12, 11, 10, 9, 8, 7, 6, 5, 4)
+);
+
+
+.grid_15 {
```

Comment:
al deze debug styles zou ik naar 1 centrale partial verhuizen. Eventueel maak je daarin een array van kleuren, die je dan met een for loop voor ieder van de relevante grid maten laat uitdelen :)